### PR TITLE
1.20 port

### PR DESCRIPTION
**Pull request by gohkenytp**

### CODE REVIEW

File: src/main/resources/META-INF/mods.toml [hunk 1]
```diff
@@ -14,13 +14,13 @@ issueTrackerURL="https://github.com/bl4ckscor3/CookieBlock/issues"
 [[dependencies.cookieblock]]
     modId="forge"
     mandatory=true
-    versionRange="[44.0.6,)"
+    versionRange="[46.0.1,)"
```
Comment (bl4ckscor3) [inline src/main/resources/META-INF/mods.toml:17]: The version is too restrictive. The mod would work on 46.0.0 as well.

File: src/main/resources/META-INF/mods.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 modLoader="javafml"
-loaderVersion="[44,)"
+loaderVersion="[46,)"
 license="MIT https://github.com/bl4ckscor3/CookieBlock/blob/1.19/LICENSE"
```
Comment (bl4ckscor3) [inline src/main/resources/META-INF/mods.toml]: Needs to be updated to 1.20.

File: build.gradle [hunk 3]
```diff
@@ -5,22 +5,22 @@ buildscript {
...
-        classpath group: 'net.minecraftforge.gradle', name: 'ForgeGradle', version: '5.1.+', changing: true
+        classpath group: 'net.minecraftforge.gradle', name: 'ForgeGradle', version: '[6.0,6.2)', changing: true
     }
 }
 
 apply plugin: 'net.minecraftforge.gradle'
 apply plugin: 'eclipse'
 
-version = '1.2.1'
+version = '1.2.2'
```
Comment (bl4ckscor3) [inline build.gradle]: No version increase for a Minecraft update.

**Review by bl4ckscor3 — CHANGES_REQUESTED** [review]
Thank you again. I have noted the same things in this PR as in your other one on the Ceiling Torch repository. Please make sure that should you want to port any of my other mods, that these issues are addressed up front. Otherwise I will not be accepting further PRs.

- Upgrade the Gradle wrapper and relevant scripts to 8.1.1. Best to just copy the files from the Forge MDK.
- Please apply this change https://github.com/Geforce132/SecurityCraft/commit/f4d2a30c73c2a64c139d329c3a6f95dfb0d1f990#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L26-R29
- Same with this one https://github.com/Geforce132/SecurityCraft/commit/f4d2a30c73c2a64c139d329c3a6f95dfb0d1f990#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R53
- Please add this https://github.com/Geforce132/SecurityCraft/commit/f4d2a30c73c2a64c139d329c3a6f95dfb0d1f990#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R139-R141

**Comment by bl4ckscor3** [comment]
Not everything is addressed yet.

**Comment by gohkenytp** [comment]
I don't want to finish this port sadly.

**Comment by gohkenytp** [comment]
Also, why are you halting progress on your port to 1.20 anyway?

**Comment by bl4ckscor3** [comment]
I am not halting progress. I am requesting the port to be done properly. I think you are severely misunderstanding how PRs work. I don't just merge changes willy-nilly. They have to actually work, which yours do not. I can't even import the mod into an IDE because you refuse to perform the changes I asked you to.

EDIT: Meant to post this in https://github.com/bl4ckscor3/CeilingTorch/pull/66 Oh well.

**Comment by gohkenytp** [comment]
Not to be rude but, I just can't do it now. OK? Accept it. Maybe you can port it without me if you get a chance.

**Comment by bl4ckscor3** [comment]
I still don't get your problem with this. I never expected you to immediately address all my comments. I don't wish to interact with you any further.

**Pull request closed without merging**